src/nanogram.ts

```diff
@@ -49,15 +49,21 @@ export default class Nanogram {
   /*
    * Make fetch call to instagram
    */
-  private async HTTP<T>(request: RequestInfo, useRegExp = true): Promise<T | undefined> {
-    const requestOptions: RequestInit = {
-      method: 'GET',
-      redirect: 'follow',
+  private async HTTP<T>(request: string, useRegExp = true): Promise<T | undefined> {
+    const xhrrequest = (method: string, url: string): Promise<XMLHttpRequest> => {
+      return new Promise(function (resolve, reject) {
```

Comment:
Can you rewrite it to an arrow function?

`js
return new Promise((resolve, reject) => {..})
`